### DISCUSSION TITLE
Constricted panels for NDS

### DISCUSSION THREAD

**Discussion by Tomlennon21** [nhsevidence/NICE-Experience#184] [General]
On the https://www.nice.org.uk/news page we have grey panels that are all constricted to be the same size. This, though is old code. Can we get the same functionality in the new design system?

**Comment by wa-rren-dev** [comment]
Well @Tomlennon21 you'll be delighted to learn that this is already in the live design system now. This pull request #159 converted the grid into a flexbox-based one, so now you can use

```
<div class="grid grid--equal-height">
 <div data-g="12 sm:4" style="background: red">These <br> will <br> be <br> the</div>
 <div data-g="12 sm:4" style="background: green">same</div>
 <div data-g="12 sm:4" style="background: blue">height</div>
</div>
```

We haven't done a good job of updating the docs, will get on that.

**Comment by Tomlennon21** [comment]
I have passed this on to the team. Thanks for that :)
